component/component/main.jsonnet

```diff
@@ -195,12 +195,48 @@ local emailSecret = kube.Secret(params.services.vshn.emailAlerting.secretName) {
   },
 };
 
+local filterName(name) = '|' + if name == 'postgres' then 'postgresql' else if name == 'emailAlerting' then '' else name;
+local backupJobRegex = std.foldl(function(prev, current) prev + filterName(current.name), common.FilterServiceByBoolean('enabled'), '');
```

Comment:
This regex currently produces strange (but probably valid) output in some cases. eg:
```
vshn-(||minio)-
```

Reply:
it allows empty value:
```
vshn--
```
is a match. that's bad. I know it's a corner case we wouldn't probably ever get, but I'd fix it just to sleep better